app/models/alias_nav_item.rb

```diff
@@ -6,14 +6,10 @@ class AliasNavItem < NavItem
   validates :title, :alias_id, :parent, presence: true
 
   crud.config do
-    fields parent_id:     { type: :hidden },
-           is_hidden:     { type: :boolean },
-           alias_id:      { type: :tree },
-           if:            { type: :code },
-           unless:        { type: :code },
-           method:        { type: :code },
-           highlights_on: { type: :code },
-           content_block: { type: :code }
+    fields parent_id: { type: :hidden },
+           is_hidden: { type: :boolean },
+           alias_id:  { type: :tree },
+           method:    { type: :code }
```

Comment:
still need this method type code?

Reply:
This controls how it displays. It doesn't need to be code but it looks better that way.